### PR TITLE
tags/instructor/course/*.tag: change to 2-space indentation #7352

### LINKED ISSUE #7352: *.jsp: change to 2-space indentation
Refactor all *.jsp pages to use 2-space indentation instead of 4.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352

### CODE REVIEW

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSessionPermissionsDiv.tag [hunk 1]
```diff
@@ -8,37 +8,37 @@
 <%@ attribute name="sectionRow" type="teammates.ui.template.CourseEditSectionRow" required="true" %>
 
 <div id="tuneSessionPermissionsDiv${panelIndex}ForInstructor${instructorIndex}" class="row" data-is-originally-displayed="${sectionRow.sessionsInSectionSpecial}"
-    <c:if test="${not sectionRow.sessionsInSectionSpecial}">
-        style="display: none;"
-    </c:if> >
-    <input type="hidden" name="is<%=Const.ParamsNames.INSTRUCTOR_SECTION_GROUP%>${panelIndex}sessionsset" value="${sectionRow.sessionsInSectionSpecial}"/>
+  <c:if test="${not sectionRow.sessionsInSectionSpecial}">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSessionPermissionsDiv.tag]: Same here :)

File: src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSessionPermissionsDiv.tag [hunk 2]
```diff
@@ -8,37 +8,37 @@
...
-                    </c:forEach>
-                </tr>
-            </c:forEach>
-        </tbody>
-    </table>
+      <c:forEach items="${sectionRow.feedbackSessions}" var="feedbackSession">
+        <tr>
+          <td>${feedbackSession.feedbackSessionName}</td>
+          <c:forEach items="${feedbackSession.permissionCheckBoxes}" var="checkbox">
+          <td class="align-center">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSessionPermissionsDiv.tag]: This needs more indentation.

Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/instructor/course/courseEditTuneSessionPermissionsDiv.tag]: Done :)

**Pull request merged**